src/components/balances/AssetsTable/useHideAssets.ts

```diff
@@ -0,0 +1,88 @@
+import useChainId from '@/hooks/useChainId'
+import useHiddenTokens from '@/hooks/useHiddenTokens'
+import { useAppDispatch } from '@/store'
+import { setHiddenTokensForChain } from '@/store/settingsSlice'
+import { useCallback, useState } from 'react'
+
+// This is the default for MUI Collapse
+export const COLLAPSE_TIMEOUT_MS = 300
```

Comment:
No need to export it, it's used in the same module.

Reply:
The test is using it.